disperser/dataapi/docs/v2/V2_docs.go

```diff
@@ -572,10 +572,6 @@ const docTemplateV2 = `{
                 "reservation_period": {
                     "description": "ReservationPeriod represents the range of time at which the dispersal is made",
                     "type": "integer"
-                },
-                "salt": {
-                    "description": "Allow same blob to be dispersed multiple times within the same reservation period",
-                    "type": "integer"
```

Comment:
Interesting. Why were these docs not updated as part of the other PRs? Are these just hand crated? If not, should we have a ci to check that they have been regenerated correctly?

Reply:
I think we should

Reply:
It may be good starter task for infra, filed ticket: https://github.com/Layr-Labs/eigenda/issues/1142